README.md

```diff
@@ -395,6 +395,8 @@ where:
      if full primary key equal condition is specified
   * `timeout` (`?number`) - `vshard.call` timeout (in seconds)
   * `fields` (`?table`) - field names for getting only a subset of fields
+  * `fullscan` (`?boolean`) - if `true` then critical log entry will be skipped
+    on empty or nil `select`
```

Comment:
`crud.pairs()` description has the following phrase:

> Its arguments are the same as `crud.select` arguments,
> but negative `first` values aren't allowed.

It should be corrected.

Reply:
Updated.
```patch
--- a/README.md
+++ b/README.md
@@ -441,8 +443,9 @@ See more examples of select queries [here.](https://github.com/tarantool/crud/bl
 ### Pairs
 
 You can iterate across a distributed space using the `crud.pairs` function.
-Its arguments are the same as [`crud.select`](#select) arguments,
-but negative `first` values aren't allowed.
+Its arguments are the same as [`crud.select`](#select) arguments except
+`fullscan`(it does not exist because `crud.pairs` does not generate a critical
+log entry on empty or nil conditions), but negative `first` values aren't allowed.
 User could pass use_tomap flag (false by default) to iterate over flat tuples or objects.
 
 **Example:**
 ```